events/event_nettrace.go

```diff
@@ -16,6 +20,18 @@ var NetTraceIntegrator = func() func(basictracer.SpanEvent) {
 			tr.SetMaxEvents(1000)
 		case basictracer.EventFinish:
 			tr.Finish()
+		case basictracer.EventTag:
+			tr.LazyPrintf("%s:%v", t.Key, t.Value)
+		case basictracer.EventLogFields:
+			var buf bytes.Buffer
+			for i, f := range t.Fields {
+				if i > 0 {
+					buf.WriteByte(' ')
+				}
+				fmt.Fprintf(&buf, "%s:%v", f.Key(), f.Value())
+			}
+
+			tr.LazyPrintf("%s", buf.String())
```

Comment:
It's a judgment call - `bytes.Buffer` is at least 100 bytes or so and would need to get moved to heap and kept around, while the average message is much shorter.